packages/react-scripts/fixtures/kitchensink/src/features/env/NodePath.js

```diff
@@ -1,21 +1,27 @@
-import React from 'react'
+import React, { Component, PropTypes } from 'react'
 import load from 'absoluteLoad'
 
-export default class extends React.Component {
-  constructor(props) {
-    super(props);
+export default class extends Component {
+  static propTypes = {
+    notifyRendered: PropTypes.func
+  }
 
-    this.done = () => {};
-    this.props.setCallWhenDone && this.props.setCallWhenDone((done) => {
-      this.done = done;
-    });
+  static defaultProps = {
+    notifyRendered: () => {}
```

Comment:
Oh, okay!

I think it would make sense to also test behavior in those tests. So that we know Jest environment doesn't just "not crash", but also works correctly.